crt_portal/cts_forms/templates/forms/pro_template.html

```diff
@@ -8,7 +8,7 @@
 {% endblock %}
 
 {% block form_questions %}
-<div class="crt-portal-card">
+<div class="crt-portal-card crt-portal-card--proform">
```

Comment:
These extra classes are just so I can target only the proform  `h2`s with CSS. Otherwise the hack for getting the headings to show would get applied to every card in the app, which seemed unnecessary.